Dockerfile.template

```diff
@@ -11,7 +11,7 @@ RUN echo "deb http://pipplware.pplware.pt/pipplware/dists/jessie/main/binary /"
 wget -O - http://pipplware.pplware.pt/pipplware/key.asc | sudo apt-key add -
 
 # WTF is going on with httpredir from debian? removing it the dirty way
-RUN sed -i "s/httpredir.debian.org/`curl -s -D - http://httpredir.debian.org/demo/debian/ | awk '/^Link:/ { print $2 }' | sed -e 's@<http://\(.*\)/debian/>;@\1@g'`/" /etc/apt/sources.list
+RUN sed -i "s@httpredir.debian.org@`curl -s -D - http://httpredir.debian.org/demo/debian/ | awk '/^Link:/ { print $2 }' | sed -e 's@<http://\(.*\)/debian/>;@\1@g'`@" /etc/apt/sources.list
```

Comment:
It might be better to use `|` or something, since it's possible the url could be `user:pass@mirror...`, although I don't think that'll ever be the case